LibraBFT/ImplShared/Consensus/Types.agda

```diff
@@ -153,26 +155,16 @@ module LibraBFT.ImplShared.Consensus.Types where
      -- RoundManager directly
   open RoundManager public
 
+  -- TODO-2: We would need dependent lenses to have a lens from RoundManager to
```

Comment:
One thing to note is that we have quite a few `GetterNoFunctor` definitions in the Haskell code.  Because these are `Getter`-only, we _can_ have the equivalent in our context.  I think this would greatly reduce the number of cases in which the Agda model looks different to the Haskell code.  Something to think about; I'm not holding up this PR for this.